src/docbuilder/rustwide_builder.rs

```diff
@@ -500,6 +504,20 @@ impl RustwideBuilder {
             .purge_from_cache(&self.workspace)
             .map_err(FailureError::compat)?;
         local_storage.close()?;
+        if let Some(distribution_id) = self.config.cloudfront_distribution_id_web.as_ref() {
+            if successful {
```

Comment:
We might want to invalidate unsuccessful builds too so they appear as broken in the version list.

Reply:
valid, changed